pkg/build/api/v1/conversion.go

```diff
@@ -105,6 +105,21 @@ func convert_api_CustomBuildStrategy_To_v1_CustomBuildStrategy(in *newer.CustomB
 	return nil
 }
 
+func convert_v1_ExternalBuildStrategy_To_api_ExternalBuildStrategy(in *ExternalBuildStrategy, out *newer.ExternalBuildStrategy, s conversion.Scope) error {
+	if err := s.DefaultConvert(in, out, conversion.IgnoreMissingFields); err != nil {
+		return err
+	}
+	return nil
+}
+
+// empty conversion needed because the conversion generator can't handle unidirectional custom conversions
+func convert_api_ExternalBuildStrategy_To_v1_ExternalBuildStrategy(in *newer.ExternalBuildStrategy, out *ExternalBuildStrategy, s conversion.Scope) error {
+	if err := s.DefaultConvert(in, out, conversion.IgnoreMissingFields); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
You shouldn't need any of this since you're just doing default conversions in both cases.  We needed it for the other strategies because they are not straight default conversions.